rep-2006.rst

```diff
@@ -0,0 +1,73 @@
+REP: 2006
+Title: ROS 2 Vulnerability Disclosure Policy
+Author: ROS 2 Security Working Group, Chris Lalancette <clalancette@openrobotics.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 12-May-2020
+Post-History:
+
+
+Abstract
+========
+
+This REP describes the Vulnerability Disclosure Policy related to the ROS 2 Common Packages as defined in `REP-2005 <https://www.ros.org/reps/rep-2005.html>`_.
+
+
+Motivation
+==========
+
+The developers of the Robot Operating System (ROS) take security very seriously.
+As such, we would like to be informed when a security bug is found so that it can be fixed and disclosed as quickly as possible.
+The rest of the document outlines what is covered by this policy and how to report security vulnerabilities.
+
+Scope
+=====
+
+The Vulnerability Disclosure Program outlined here covers all code within the ROS 2 Common Packages as defined in REP 2005.
+
+The ROS Security Working Group may also be able to assist you with reporting vulnerabilities which are not in scope of this policy but which are related to ROS or more generally to robotics.
+An increasing number of companies are bringing ROS-based products to market, and during this growth period we anticipate vulnerabilities to be identified in products before vendors have an established vulnerability reporting program.
+We will work with you on a best-effort basis to help connect you with responsible parties best suited to address your concerns.
+
+How to submit a vulnerability
+=============================
+
+Please submit vulnerability reports by emailing security@openrobotics.org, preferably in English.
+This is a private list of security officers who can help verify the bug report and develop and release a fix.
+The more information provided about the bug, the easier it will be to fix.
+If you already have a fix, please include it with your report.
```

Comment:
Should we specify a favored format for the fix (e.g. git patch)? Similarly to the favored language for the mail.

Reply:
+1 I just mentioned this in my comment as well

Reply:
My opinion is that adding too much structure at the beginning can scare people off, and isn't really necessary.  I'm going to (perhaps naively) assume that security researches know how to send a patch.  If we start consistently getting h.264-encoded patches or something, we can amend this to add more structure.

Reply:
Is this only designed for established pentesters? What if some junior-dev suddenly discovers a behavior of its ROS2 robot that he considers security relevant and wants to escalate to the security WG? I would suggest not adding a strong typed format but maybe a suggestion for "newbies". Therefor, it would be easier to flag false positives and kindly point out a wrong config or so.

Reply:
I don't really see the situation as being much different for junior people.  If they don't know how to send a patch, then we'll get something that is maybe not useful.  But my opinion doesn't change here; unless/until this becomes a common occurrence, I don't really see a need.  Maybe people in the security working group disagree.

Reply:
As per [my comment earlier](https://github.com/ros-infrastructure/rep/pull/262#issuecomment-632755837) this is something we are going to discuss during the next WG meeting - not the patch format but if we should set a report template at all.